editors/tea/Portfile

```diff
@@ -61,8 +73,10 @@ variant qt5 conflicts qt4 description {Use Qt 5} {}
 
 if {[variant_isset qt4]} {
     PortGroup           qmake 1.0
+
```

Comment:
Why the extra carriage return?

Reply:
Port lint wants an empty line after PortGroup.

Reply:
"port lint" demands lots of silly things; we need to fix it. It spews insane amounts of garbage for all rust ports for example, and this is another issue (it should not insist on a blank line in this case.)